container_images/gce-img-resource/check/check_command.go

```diff
@@ -39,10 +39,19 @@ func Run(request Request) (Response, error) {
 	}
 
 	call := computeService.Images.List(request.Source.Project)
+
+	var filter string
+
+	if request.Source.ReadyOnly {
```

Comment:
Can you update the comment starting on line 13?

Reply:
Done